distributed/tests/test_client.py

```diff
@@ -7345,6 +7336,109 @@ async def test_dump_cluster_state_json(c, s, a, b, tmp_path, local):
         await c.dump_cluster_state(filename, format="json")
 
 
+def blocked_inc(x, event):
+    event.wait()
+    return x + 1
+
+
+@pytest.mark.parametrize("local", [True, False])
+@pytest.mark.parametrize("_format", ["msgpack", "yaml"])
```

Comment:
Probably not important to parameterize over these (especially local). We should be able to trust that other tests are adequately checking that cluster dumps are created correctly; this test can just focus on reading one.

Reply:
Removed excess parametrisation